source/common/json/json_loader.h

```diff
@@ -1,10 +1,17 @@
 #pragma once
 
 #include <list>
+#include <map>
+#include <stack>
 #include <string>
 
 #include "envoy/json/json_object.h"
 
+// Do not let RapidJson leak outside json_loader.
```

Comment:
I guess this can't be moved into a header/cc file. Sorry I didn't see the rapidjson::Document in your member variables earlier.